lib/menu.rb

```diff
@@ -0,0 +1,34 @@
+
+
+class Menu
+
+  attr_reader :food, :selection
+
+  def initialize
+    @selection = {}
```

Comment:
As you define `@selection` in `choose_items` method, do you need to initialize with it?